MinGW warning: inline function declared as dllimport: attribute ignored [-Wattributes]

This is with GCC 12.2.0 on [MSYS2 UCRT64](https://github.com/msys2/MINGW-packages/actions/runs/3417550887/jobs/5688814001#step:11:319)/MINGW64/MINGW32:

```
  In file included from C:/_/mingw-w64-imath/src/Imath-3.1.6/src/python/PyImath/PyImathFixedArrayTraits.h:13,
                   from C:/_/mingw-w64-imath/src/Imath-3.1.6/src/python/PyImath/PyImathBufferProtocol.cpp:12:
  C:/_/mingw-w64-imath/src/Imath-3.1.6/src/Imath/half.h:623:37: warning: inline function 'constexpr uint16_t Imath_3_1::half::bits() const' declared as dllimport: attribute ignored [-Wattributes]
    623 |     IMATH_EXPORT constexpr uint16_t bits() const IMATH_NOEXCEPT;
        |                                     ^~~~
  C:/_/mingw-w64-imath/src/Imath-3.1.6/src/Imath/half.h:626:41: warning: inline function 'constexpr void Imath_3_1::half::setBits(uint16_t)' declared as dllimport: attribute ignored [-Wattributes]
    626 |     IMATH_EXPORT IMATH_CONSTEXPR14 void setBits (uint16_t bits) IMATH_NOEXCEPT;
        |                                         ^~~~~~~
```

--- a/src/Imath/half.h
+++ b/src/Imath/half.h
@@ -619,10 +619,10 @@ public:
     /// @name Access to the internal representation
 
     /// Return the bit pattern
-    IMATH_EXPORT constexpr uint16_t bits () const IMATH_NOEXCEPT;
+    constexpr uint16_t bits () const IMATH_NOEXCEPT;
 
     /// Set the bit pattern
-    IMATH_EXPORT IMATH_CONSTEXPR14 void setBits (uint16_t bits) IMATH_NOEXCEPT;
+    IMATH_CONSTEXPR14 void setBits (uint16_t bits) IMATH_NOEXCEPT;
 
     /// @}
 

